runtime/interpreter/statictype.go

```diff
@@ -314,6 +330,21 @@ func NewConstantSizedStaticType(
 	}
 }
 
+func (t ConstantSizedStaticType) IsComposite() bool {
+	return false
+}
+
+func (t ConstantSizedStaticType) Copy() atree.TypeInfo {
+	// TODO: is creating a new ConstantSizedStaticType necessary?
+	// atree needs immutable TypeInfo or a copy of mutable TypeInfo because TypeInfo can be shared.
+	return t
```

Comment:
Same as above: this might only do a shallow-copy since certain types (e.g: `RestrictedStaticType`) have pointer receivers.